RELEASE.md

```diff
@@ -5,5 +5,6 @@ The Kudo Project is released on an as-needed basis. The process is as follows:
 1. An issue is proposing a new release with a changelog since the last release
 2. All [OWNERS](OWNERS) must LGTM this release
 3. An OWNER runs `git tag -s $VERSION` and inserts the changelog and pushes the tag with `git push $VERSION`
-4. The release issue is closed
-5. An announcement email is sent to `kubernetes-kubebuilder@googlegroups.com` with the subject `[ANNOUNCE] Kudo $VERSION is released`
+4. The corresponding docker image `kudobuilder/controller:$VERSION` is pushed to dockerhub
```

Comment:
Is that version captured in the REPO somewhere?

Reply:
No, should we add something like “Define $VERSION according to [semantic versioning](https://semver.org/)” ?

Reply:
Maybe move that discussion to another issue? I feel we could define it, but that would entail changes to the Make/Dockerfile which seems orthogonal to this PR/